Extensions/TileMap/tilemapruntimeobject-pixi-renderer.ts

```diff
@@ -26,8 +26,20 @@ namespace gdjs {
       this._object = runtimeObject;
       this._runtimeScene = runtimeScene;
 
+      const pixiRenderer = runtimeScene
+        .getGame()
+        .getRenderer()
+        .getPIXIRenderer();
+
+      PIXI.tilemap.settings.use32bitIndex = true;
```

Comment:
This is what fix the tiles limit. It takes a bit more memory but it should be fine.

Reply:
maybe adding a comment above this line, and the one in `JSExtension` to say why it's important to keep. 
Something like "This is required to allow a high number of tiles"

Reply:
Yep, let's add comments on both lines :)